bloodytools/simulation_objects/simulation_objects.py

```diff
@@ -1237,6 +1213,18 @@ def simulate_with_raidbots(self, apikey) -> bool:
 
     return simc_hash
 
+  def set_json_data(self, data: dict):
```

Comment:
`def set_json_data(self, data: dict) -> None:`

Reply:
I'm sure return type predictions are missing elsewhere in my code, too. But I think it's good to add it whenever something like this is created or edited.